LogicalTypes.md

```diff
@@ -655,10 +655,12 @@ should always be determined by the following rules:
 1. If the repeated field is not a group, then its type is the element type and
    elements are required.
 2. If the repeated field is a group with multiple fields, then its type is the
-   element type and elements are required.
-3. If the repeated field is a group with one field and is named either `array`
-   or uses the `LIST`-annotated group's name with `_tuple` appended then the
-   repeated type is the element type and elements are required.
+   element type and elements are required. In this case, the element type is
+   a Struct type with multiple fields.
+3. If the repeated field is a group (without annotation) with one `required` or
```

Comment:
I doesn't hurt to make it explicit.

> I'm also wondering if we should add an example above for the non-LIST annotated repeating fields, or should that be a new PR?

I think we should, but I am not clear where to add it since these lines are all talking about LIST-annotated structures.

Reply:
> I think we should, but I am not clear where to add it since these lines are all talking about LIST-annotated structures.

Perhaps around line 575 where the unannotated form is introduced. 🤷  I admit this is a bit of an awkward location, since this file is about logical types. Out of scope here, but perhaps a new markdown file that documents the various ways repetition is supported would be appropriate. I know it would help me.

Reply:
I think we'd better open another PR to discuss list types without annotation. It might be more complicated than the scope of this PR. I'm not sure if it should be allowed by the spec even though we are able to create the schema.